### PR TITLE
Create final report.adoc

### PR DESCRIPTION
**Pull request by wkwong-ribose**
This is the draft of the final report for Metanorma GSoD 2021.
It will be published on the Metanorma website, after the review.

### CODE REVIEW

File: final report.adoc [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+|===
+
+=== Results
+
+Details about the results can be found https://docs.google.com/presentation/d/1vLfafPX3gAakUwV8P5VMpqsGxupZ3WBkFGZY1J4V-fI/edit#slide=id.p1[here].
+
+The team carried out both quantitative audit and qualitative audit. 94 sources were analyzed (including website entries and slide decks). 
+Problems were identified and summarized in the https://github.com/metanorma/gsod/tree/master/artifacts/Analysis[analysis reports]. An information architecture was developed to tackle the problems.
+A tutorial for new users was created and some existing documentation was restructured.
+
```
Comment (techtrailhead) [inline final report.adoc:145]: Comprehensive guides were created for ISO and OGC flavors along with cheat sheets for quick use by the users.

**Review by techtrailhead — COMMENTED** [review]
Overall I liked the preciseness of the report. Only small language change–we should write the report in the past tense like using 'was' instead of 'is'. I have added one more point as a comment which I guess summarizes the entire documentation change.

File: final report.adoc (hunk 1 above)
Comment (techtrailhead) [reply final report.adoc:145]: Also IETF

Comment (kickoke) [reply final report.adoc:145]: Added :)

**Review by kickoke — APPROVED** [review]
I added my feedback as a commit and incorporated Ankita's notes as well.

**Pull request merged**